CMakeLists.txt

```diff
@@ -51,7 +51,7 @@ set(CMAKE_CXX_FLAGS
 	-Werror=uninitialized
 	-Werror=init-self
 	-Wcast-qual
-	-Wconversion
+	-Wno-conversion
```

Comment:
I did not want to cast every single constant in my test code from double
(since my system uses doubles by default) to float. My reasoning is that
most embedded systems use float so unexpected automatic casts from double
to float are rarely going to be a problem in practice. I don't think extra
casts are ever a problem either because compiler should be smart enough to
optimize that.

On 16 Aug 2016 18:09, "James Goppert" <notifications@github.com> wrote:

> In CMakeLists.txt
> <https://github.com/PX4/Matrix/pull/26#discussion_r74966488>:
>
> > @@ -51,7 +51,7 @@ set(CMAKE_CXX_FLAGS
> >  	-Werror=uninitialized
> >  	-Werror=init-self
> >  	-Wcast-qual
> > -	-Wconversion
> > +	-Wno-conversion
>
> Why did you turn of conversion warning, this is important for the embedded
> sign to force user to realize if he is wasting casts.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/PX4/Matrix/pull/26/files/3a1e1e3d2bdfaebd9efd87bdeb9d135dfe6d4453#r74966488>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AERqiSRUZLeIUnZ1hYeFwnfEMZ-2unk5ks5qgeDSgaJpZM4JZZwu>
> .
>